nodes/vr_sros/vr-sros.go

```diff
@@ -84,6 +85,29 @@ func (s *vrSROS) PreDeploy(_ context.Context, _, _, _ string) error {
 	return createVrSROSFiles(s)
 }
 
+func (s *vrSROS) PostDeploy(_ context.Context, _ map[string]nodes.Node) error {
+	// if we got a partial config, it's time to load and apply it
+	if isPartialConfigFile(s.Cfg.StartupConfig) {
+		// read the file
+		bcfg, err := utils.ReadFileContent(s.Cfg.StartupConfig)
+		if err != nil {
+			return err
+		}
+		// convert []byte to string config
+		cfg := string(bcfg)
+		// replace \r\n by \n and then split by \n
+		cfgSlc := strings.Split(strings.ReplaceAll(cfg, "\r\n", "\n"), "\n")
+		// apply config to node
+		log.Infof("Applying partial config %s to %s", s.Cfg.StartupConfig, s.Cfg.LongName)
+		err = netconf.ApplyConfig(s.Cfg.MgmtIPv4Address, scrapliPlatformName, defaultCredentials.GetUsername(), defaultCredentials.GetPassword(), cfgSlc)
+		if err != nil {
+			return err
+		}
+		log.Infof("%s successfully configured", s.Cfg.LongName)
+	}
+	return nil
+}
+
```

Comment:
I believe scrapli has a function (or option) that send configs from file already. So we can use that one, instead of reading the file ourselves.